src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Utilities/BKTree.cs

```diff
@@ -63,14 +55,15 @@ private BKTree(char[] concatenatedLowerCaseWords, ImmutableArray<Node> nodes, Im
             _edges = edges;
         }
 
-        public static BKTree Create(params string[] values)
-            => Create((IEnumerable<string>)values);
-
         public static BKTree Create(IEnumerable<string> values)
             => new Builder(values).Create();
 
         public void Find(ref TemporaryArray<string> result, string value, int? threshold = null)
         {
+            Contract.ThrowIfNull(value, nameof(value));
+            Contract.ThrowIfNull(_concatenatedLowerCaseWords, nameof(_concatenatedLowerCaseWords));
+            Contract.ThrowIfTrue(_nodes.IsDefault, $"{nameof(_nodes)}.{nameof(_nodes.IsDefault)}");
+            Contract.ThrowIfTrue(_edges.IsDefault, $"{nameof(_edges)}.{nameof(_edges.IsDefault)}");
```

Comment:
this is the important part.  ideally so we get a dump that tells us which of these is the problem.

Reply:
Is this going to stay in the PR since you think you've figured out the problem?

Reply:
yeah, i don't mind it :)